index.rst

```diff
@@ -249,6 +344,11 @@ What are the open questions that drive the differences between different viable
 
 - Can we feed FGCM with compensated-filter photometry in order to run it before background estimation is complete?
 
+- Do we need some kind of coaddition or warp-comparison to finalize our per-visit background models?  If so, will those also impact PSF modeling and/or aperture corrections?
+
+- Is a second round of astrometric fitting necessary to ensure PSF chromaticity is consistent with our (achromatic) WCSs?  Is it sufficient?
```

Comment:
Yes.  Putting DCR in the WCS is both really problematic (you can't warp images to the same pixel grid for coaddition if the WCS depends on sub-band SED) and at best an approximation (it's not a shift, it's an asymmetric smearing, unless the SED is a delta function), while it can be fully corrected by having a chromatic PSF.